### DISCUSSION TITLE
Thumbnails in the presenter view

### DISCUSSION THREAD

**Discussion by mil-ad** [marp-team/marp#372] [Q&A] [answered]
Is it possible to get a thumbnail video of all slides in the presenter view so that you can jump directly to specific slides? Could be useful for example during post-presentation Q&A when you want to go back to particular slides directly.

**Comment by yhatt** [comment, marked as answer]
Potentially duplicated with #326 and https://github.com/marp-team/marp-cli/issues/464.
